src/rgw/rgw_file.h

```diff
@@ -248,33 +248,38 @@ namespace rgw {
     friend class RGWLibFS;
 
   private:
-    RGWFileHandle(RGWLibFS* _fs, uint32_t fs_inst)
+    RGWFileHandle(RGWLibFS* _fs)
       : fs(_fs), bucket(nullptr), parent(nullptr), variant_type{directory()},
 	depth(0), flags(FLAG_ROOT)
       {
 	/* root */
 	fh.fh_type = RGW_FS_TYPE_DIRECTORY;
 	variant_type = directory();
 	/* stat */
-	state.dev = fs_inst;
 	state.unix_mode = RGW_RWXMODE|S_IFDIR;
 	/* pointer to self */
 	fh.fh_private = this;
       }
 
+    uint64_t init_fsid(std::string& uid) {
```

Comment:
I'm confused a bit here;  at line 276, we call it with "fsid" which is the name of the argument;  does this say that fsid == user name?  I do think that was the original intent of fs_inst() so logically it seems correct